compiler/src/dotty/tools/dotc/transform/Erasure.scala

```diff
@@ -626,17 +626,19 @@ object Erasure extends TypeTestsCasts{
 
                     if (isRequired) {
                       // check for clashes
-                      val clash: Option[Symbol] = oldSymbol.owner.info.decls.lookupAll(bridge.name).find {
-                        sym =>
-                          (sym.name eq bridge.name) && sym.info.widen =:= bridge.info.widen
-                      }.orElse(
-                        emittedBridges.find(stat => (stat.name == bridge.name) && stat.tpe.widen =:= bridge.info.widen)
+                      val clash: Option[Symbol] = oldSymbol.owner.info.memberExcluding(bridge.name, Flags.Bridge).alternatives.find {
+                        denot => {
+                          !denot.symbol.owner.is(Flags.Trait) && (denot.symbol.owner ne defn.ObjectClass) &&
```

Comment:
All these conditions where added to make the compilation of the standard library work properly. I will have to explore in detail why I had to exclude traits and object. The non trivial subclass of `bridge.owner` are present in `scala.collection`. Nevertheless I will add a small test that has non-trivial subclass.